Dockerfile

```diff
@@ -17,7 +17,5 @@ RUN yum install -y git && \
     yum remove -y apache-maven && \
     rm /etc/yum.repos.d/epel-apache-maven.repo
```

Comment:
Could we make this

```
    rm /etc/yum.repos.d/epel-apache-maven.repo && \
    touch /opt/sonatype/nexus/etc/crowd.properties && \
    chown nexus /opt/sonatype/nexus/etc/crowd.properties && \
    chmod u+w /opt/sonatype/nexus/etc/crowd.properties
```

(or something like that), and continue running as `nexus`?